### DISCUSSION TITLE
Good way of syncing shop configuration accross development environments

### DISCUSSION THREAD

**Discussion by ocelot2123** [saleor/saleor#8487] [Deployment]
Hi, just wondering does anyone have a good recommendation on how to sync configuration across multiple environments.

Like for example, I will add a warehouse, or like a shipping zone, how can I easily have that synced out to all other developers so they end up with the same shop configuration as me?

**Comment by karolkielecki** [comment]
@ocelot2123 Hi! 

The first option would be to prepare everything suitable on your DB, then create a DB dump and send it to other devs to restore. Using this option, they will have the same data as you do.

The 2nd option is to modify populatedb script - it contains a lot of default data, you can just extend it and tell other devs to run it later.

Cheers,

**Reply by ocelot2123** [reply]
yea thats basically what we are currently doing. The main problem I have is you have to delete you DB first before you restore it otherwise it will show conflicts which is kinda annoying if you have existing data and whatnot and was hoping there would be a better way.
